packages/react-core/src/use-module.ts

```diff
@@ -1,25 +1,15 @@
 import { useEffect, useState, useCallback } from 'react';
-import { asyncLoader, getCachedModule, ExposedScalprumModule } from '@scalprum/core';
+import { getCachedModule, ExposedScalprumModule, getScalprum } from '@scalprum/core';
 
-export function useModule(
-  scope: string,
-  module: string,
-  defaultState: any,
-  options: {
-    skipCache?: boolean;
-  } = {}
-): ExposedScalprumModule | undefined {
-  const defaultOptions = {
-    skipCache: false,
-    ...options,
-  };
+export function useModule(scope: string, module: string, defaultState: any): ExposedScalprumModule | undefined {
```

Comment:
FYI @florkbr @spadgett @christianvogt

Scalprum has a `useModule` hook that provides access to a specific exposed module of the given plugin.

This is basically a React hook equivalent of `PluginStore.getExposedModule` function.

We can consider introducing such hook directly in the OpenShift plugin SDK.